### PR TITLE
updates in relvals 

### PR DESCRIPTION
**Pull request by prebello**
this PR aims to:

- include the new premix relval SMS-T1tttt_mG1-1500_mLSP-100_13
- adjust few jobs splitting and 
- update the regenerated GENSIM from 9_3_0_pre5

### CODE REVIEW

**Comment by fabozzi** [comment]
please test

File: Configuration/PyReleaseValidation/python/MatrixInjector.py [hunk 1]
```diff
@@ -211,6 +211,7 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['HLTDR2_50ns']=1
             wmsplit['HLTDR2_25ns']=1
             wmsplit['HLTDR2_2016']=1
+            wmplit['HLTDR2_2017']=1
```
Comment (fabozzi) [inline Configuration/PyReleaseValidation/python/MatrixInjector.py]: @prebello 
please correct the typo in this line: wmplit -> wmsplit

**Comment by fabozzi** [comment]
please test

**Comment by fabozzi** [comment]
+1

**Comment by fabozzi** [comment]
@kpedro88 @davidlange6 
just a reminder for this to be signed off and merged for next release
Thanks!

**Comment by kpedro88** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**